case-lib/hijack.sh

```diff
@@ -23,8 +23,8 @@ function exit()
         sleep 1s
     fi
     # when case ends, store kernel log
-    if [ $DMESG_LOG_START_LINE -ne 0 ]; then
-        tail -n +$DMESG_LOG_START_LINE /var/log/kern.log |cut -f5- -d ' ' > $LOG_ROOT/dmesg.txt
+    if [ "$DMESG_LOG_START_LINE" -ne 0 ]; then
```

Comment:
```suggestion
    if [[ -n "$DMESG_LOG_START_LINE" && "$DMESG_LOG_START_LINE" -ne 0 ]]; then
```

Reply:
@Bin-QA can "&&" be used in bash test?  maybe use `-a` for and?

Reply:
> @Bin-QA can "&&" be used in bash test? maybe use `-a` for and?

@aiChaoSONG   In [[..]], we need to use && instead of -a.

Reply:
@keqiaozhang thanks, Please help to fix this, then we can close #175

Reply:
Updated.